src/FileSystem-Disk/FilePluginPrims.class.st

```diff
@@ -25,6 +25,38 @@ FilePluginPrims >> close: id [
 	
 ]
 
+{ #category : #'file primitives' }
+FilePluginPrims >> connectToFile: filePointer writable: writableFlag [
+	"Open the file with the supplied FILE* pointer, and return the file ID obtained.
+	writeableFlag indicates whether to allow write operations and must be compatible with the way the file was opened.
+	It is the responsibility of the caller to coordinate closing the file."
+
+	<primitive: 'primitiveConnectToFile' module: 'FilePlugin' error: error>
+	error = #'bad argument' ifTrue: [ 
+		filePointer isBytes ifFalse: 
+			[ ^self error: 'openFileFile:writable: filePointer must be bytes' ].
```

Comment:
I think prefix with method name is not needed for such errors.
Also it still uses old method name

Reply:
Hi Dennis,

Thanks for your review.

> I think prefix with method name is not needed for such errors.

I agree that strictly it is not needed.  I just find errors "easier" to understand with the additional information (but see below)...

> Also it still uses old method name

Oops.

I'll update the PR to remove the method name prefix.